src/lib/formats/uef_cas.cpp

```diff
@@ -249,29 +249,31 @@ static int16_t* uef_cas_fill_bit( uint8_t loops, int16_t *buffer, bool bit )
 
 static int uef_cas_fill_wave( int16_t *buffer, int length, const uint8_t *bytes )
 {
+	const uint8_t *b = bytes;
 	if ( bytes[0] == 0x1f && bytes[1] == 0x8b ) {
 		if ( gz_ptr == nullptr ) {
 			return 1;
 		}
-		bytes = gz_ptr;
+		b = gz_ptr;
```

Comment:
That's to support this. Or am I completely dumb?

Reply:
You don’t need the extra `b` because you can assign to `bytes` since the pointer is not `const`, only what it points to is `const`.

See, this is why `uint8_t const *` is clearer than `const uint8_t *`, because you can follow the simple rule that the `const` applies to the thing immediately to the left.

* `uint8_t const *` – the `const` applies to the `uint8_t` part, the pointer is not `const`.
* `uint8_t *const` – the `const` applies to the pointer, the `uint8_t it points to is not `const`.
* `uint8_t const *const` – the first `const` applies to the `uint8_t`, the second `const` applies to the pointer.